src/server/scripts/Northrend/IcecrownCitadel/boss_lady_deathwhisper.cpp

```diff
@@ -943,25 +938,25 @@ class npc_darnavan : public CreatureScript
                     {
                         case EVENT_DARNAVAN_BLADESTORM:
                             DoCast(SPELL_BLADESTORM);
-                            _events.ScheduleEvent(EVENT_DARNAVAN_BLADESTORM, urand(90000, 100000));
+                            _events.ScheduleEvent(EVENT_DARNAVAN_BLADESTORM, randtime(Seconds(90), Seconds(100)));
                             break;
                         case EVENT_DARNAVAN_CHARGE:
                             _canCharge = true;
                             break;
                         case EVENT_DARNAVAN_INTIMIDATING_SHOUT:
                             DoCast(SPELL_INTIMIDATING_SHOUT);
-                            _events.ScheduleEvent(EVENT_DARNAVAN_INTIMIDATING_SHOUT, urand(90000, 120000));
+                            _events.ScheduleEvent(EVENT_DARNAVAN_INTIMIDATING_SHOUT, randtime(Seconds(90), Minutes(2)));
                             break;
                         case EVENT_DARNAVAN_MORTAL_STRIKE:
                             DoCastVictim(SPELL_MORTAL_STRIKE);
-                            _events.ScheduleEvent(EVENT_DARNAVAN_MORTAL_STRIKE, urand(15000, 30000));
+                            _events.ScheduleEvent(EVENT_DARNAVAN_MORTAL_STRIKE, randtime(Seconds(15), Seconds(30)));
```

Comment:
EventMap overload means you no longer need `randtime` for scheduling. Also, this should be `.Repeat`.